sdk/typescript/README.md

```diff
@@ -319,7 +319,7 @@ const provider = new JsonRpcProvider();
 
 const devnetNftFilter = {
   All: [
-    { EventType: 'MoveEvent' },
+    { EventType: <EventType>'MoveEvent' },
```

Comment:
We shouldn't use this form of casting, and should instead use `as EventType` (for a few reasons, but basically this form of casting works inconsistently in TS and exists for legacy purposes). But I think the better change here would be to inline object and avoid the `devnetNftFilter` variable altogether:

```typescript

const devNftSub = await provider.subscribeEvent(
  {
    All: [
      { EventType: 'MoveEvent' },
      { Package: '0x2' },
      { Module: 'devnet_nft' },
    ],
  },
  (event) => {
    // handle subscription notification message here
  },
);
```

This will also fix the typescript error observed, and is a better pattern anyway as it makes error messages more understandable.

Reply:
Thanks for the feedback will change this accordingly.

Reply:
@Jordan-Mysten  example updated